Changes (#107)
canonical/brand-store-documentation: docs/tutorial/create-ubuntu-core-image.rst

    :dir: ~/{{CUSTOMER_STORE_PREFIX}}
    :input: snapcraft

The sample "product_serial" is generated by ``date -Is`` in this gadget's
``snap/hooks/prepare-device`` hook. In production the serial number should
be derived from a value inserted during the factory process or from a unique

> hardware identifier for uniqueness and traceability. See an example `here <https://documentation.ubuntu.com/dedicated-snap-store/how-to/creating-serial-numbers/>`_.

Reviewer: ```suggestion
hardware identifier for uniqueness and traceability. See `Create serial numbers <https://documentation.ubuntu.com/dedicated-snap-store/how-to/creating-serial-numbers/>`_.
```

Reviewer: Should fix broken generation.

account would be one with the **Viewer** role in the ``{{CUSTOMER_STORE_NAME}}``
and ``{{CUSTOMER_DEVICEVIEW_NAME}}`` stores.

> Log in to the web dashboard as ``{{CUSTOMER_ADMIN_EMAIL}}``, the **Reviewer**

Reviewer: ```suggestion
Log into the web dashboard as ``{{CUSTOMER_ADMIN_EMAIL}}``, the **Reviewer**
```

Other grammar fixes are great - but this one is correct as is. "Into" for entry, "in to" when it's "to <action>".

> X walked into the building.

> X came in to see me.

Author: Usually I'd agree with you but I believe this came from our own internal style guide which indicates logging into a website is "log in to" 🙃
